content/W2016/W16-event-graphics-workshop.md

```diff
@@ -0,0 +1,20 @@
+Title: Graphics Workshop
+Date: 2016-03-01
+Category: Events
+Tags: workshop, graphics
+Slug: graphics-workshop-w16
+Author: Aurora Huang
+Summary: Express your creativity by making art on 3D animation software!
+
+Come out to WiCS graphics workshop! Express your creativity and have fun with your friends by making art on an animation software! No experience is required, a female instructor will give a Blender tutorial that walks you through step-by-step to create a simple 3D animation. Female mentors will also be present to provide guidance.
+
+For this event you will need to bring your own computer and also have the latest Blender downloaded and installed: [https://www.blender.org/download/] (https://www.blender.org/download/)
+
+
+
+## Event Details ##
+
++ **Who:** Women and Non-binary individuals
```

Comment:
This is inconsistent with how we normally advertise women-only events. Can you copy our usual description?